Source/DafnyCore/Resolver.cs

```diff
@@ -2846,7 +2846,7 @@ public class ModuleBindings {
       }
 
       // ---------------------------------- Pass 1 ----------------------------------
-      // This pass:
+      // This pass or phase:
```

Comment:
I keep forgetting that these are called "pass", I'm always looking for "phase". Next time, if I forget again, I will be able to find them.